### ISSUE TITLE
Re-enable 32-bit deployments

### ISSUE THREAD

**Issue by pranavkm** [aspnet/Hosting#949]
Specifying RuntimeArchitecture.x86 fails due to https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/DeploymentParameters.cs#L38-L41. The underlying issue might have been addressed since this check was added.

**Comment by muratg** [comment]
@JunTaoLuo could you take a look?

**Comment by muratg** [comment]
Note to self: check current status.

**Comment by muratg** [comment]
@JunTaoLuo @Tratcher do we still not do 32 bit?

**Comment by JunTaoLuo** [comment]
Right, we still are not testing with 32-bit deployments: https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/DeploymentParameters.cs#L40

**Comment by muratg** [comment]
Thanks. Let's look into this after the previews.
